rest/doc.go

```diff
@@ -23,17 +23,19 @@ Supported endpoints:
 	/hover
 	/signatureHelp
 
-URL Query Parameters:
+All endpoint are only available through the HTTP method POST. For each request, you have to provide the following JSON:
 
-	expr  : A PromQL expression.
-	limit : (optional, only for /diagnostics and /completion endpoints) The maximum number of diagnostic messages returned.
-	line  : (only for /signatureHelp, /hover and /completion endpoints) The line (0 based) for which the metadata is queried.
-	char  : (only for /signatureHelp, /hover and /completion endpoints) The column (0 based) for which the metadata is queried. Characters are counted as UTF16 Codepoints.
+{
+  "expr": "a PromQL expression" # Mandatory for all available endpoints
+  "limit": 45  # Optional. It will be used only for the endpoints /diagnostics and /completion. It's the maximum number of result returned.
```

Comment:
```suggestion
  "limit": 45  # Optional. It will be used only for the endpoints /diagnostics and /completion. It's the maximum number of results returned.
```